jcstress-core/src/main/java/org/openjdk/jcstress/JCStress.java

```diff
@@ -85,6 +85,14 @@ public void run() throws Exception {
         parseResults();
     }
 
+    private TimeBudget getTimeBudget(ConfigsWithScheduler config) {
+        if (config == null) {
```

Comment:
By returning null you'll get an NPE when `printOn` is called in next line. If `config == null` you will get an NPE now. Seems like this change does not change anything. So, without that, I don't see a reason to wrap the TimeBudget instantiation in a private method.